selfdrive/test/test_routes.py

```diff
@@ -442,6 +442,10 @@
     'carFingerprint': VOLKSWAGEN.SKODA_KODIAQ_MK1,
     'enableCamera': True,
   },
+  "66e5edc3a16459c5|2021-05-23--23-44-46": {
```

Comment:
need to look into why CI passed, but this route isn't uploaded

Reply:
make sure the user has the upload raw toggle on

Reply:
I noticed that. Not only that, but it passed on an earlier commit where I forgot to merge the route in entirely, which should have failed immediately.